src/modules/app/fetchData.js

```diff
@@ -28,19 +26,7 @@ async function fetchData<P, T> (
     // Fetch if the data is not valid anymore or it hasn't been fetched yet
     dispatch(startFetchAction(endpoint.stateName, formattedUrl))
 
-    if (errorOverride) {
-      const payload = new Payload(false, formattedUrl, null, errorOverride)
-      dispatch(finishFetchAction(endpoint.stateName, payload))
-      return payload
-    }
-    if (responseOverride) {
-      const data = endpoint.mapResponse(responseOverride, params)
-      const payload = new Payload(false, formattedUrl, data, null)
-      dispatch(finishFetchAction(endpoint.stateName, payload))
-      return payload
-    }
-
-    const payload = await endpoint.request(params, formattedUrl)
+    const payload = await endpoint.request(params)
```

Comment:
Whouls we create a task to remove the errorOverride functionality?

Reply:
I think it is  not used in the RN app. Not sure though.

Reply:
This is all a duplicate of code present in the api-client. Therefore I thought I'd just remove it.

Reply:
It is used for some tests, so I don't think we should remove the functionality entirely.